category_theory/walking.lean

```diff
@@ -0,0 +1,94 @@
+-- Copyright (c) 2017 Scott Morrison. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Stephen Morgan, Scott Morrison
+
+import category_theory.functor
+
+open category_theory
+
+namespace tactic
```

Comment:
I guess these lines shouldn't stay in this file. This is also generalised one of your other PR's, not?

Reply:
I'm guessing you're talking about the `case_bash` tactic?

It is essential for the proofs that follow, and not used or generalised elsewhere. I'm not convinced that simple tactics really need to be segregated from their point of use, but if people would prefer that I can move this to `tactics/`.